src/pages/subreddit/Subreddit.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useEffect, useState} from 'react';
+import {useParams} from "react-router-dom";
+import axios from "axios";
+import {Link} from "react-router-dom";
+import './Subreddit.css'
+import Header from "../../components/header/Header";
+import numberFormat from "../../helpers/numberFormat";
+
+function Subreddit() {
+  const [subredditPost, setSubredditPost] = useState({});
```

Comment:
Ik de default waarde aanpassen naar `undefined`. Dan heb je een duidelijker onderscheid tussen of het al wél of niet is ingeladen. En daarnaast wordt je check hier beneden makkelijker
```suggestion
  const [subredditPost, setSubredditPost] = useState();
```